### DISCUSSION TITLE
How to exclude other directories from not participating in document compilation

### DISCUSSION THREAD

**Discussion by ZengTianShengZ** [vuepress/core#354] [Q&A] [answered]
I put vuepress in the root directory. There are many subdirectories under the root directory. By default, vuepress will use the .md in all directories to participate in document construction. But I only need the xx directory, is there a configuration like `ignore` to exclude the folder

```
.vuepress
-f1 // i need
-f2 // ignore
-f3 // ignore
```

**Comment by meteorlxy** [comment, marked as answer]
https://v2.vuepress.vuejs.org/reference/config.html#pagepatterns
